### ISSUE TITLE
bug: Ionic deployed on mobile website doesn't hide address bar when scrolling down

### ISSUE THREAD

**Issue by isaiahdahl** [ionic-team/ionic-framework#23260]
# Bug Report

**Ionic version:**
5.3.1
Angular 11

**Current behavior:**
Ionic deployed on mobile website doesn't hide address bar when scrolling down.

This issue keeps getting reported, people express their frustration with it, and then it dies. This needs to be addressed to be a platform worth using to deploy anything accessible by a browser!

**Expected behavior:**
Like every other website on the internet, when you scroll down on Chrome or Safari, the browser doesn't hide the address bar.  This is an ugly experience, and with browsers designed specifically so that they hide on scroll we lose so much screen real-estate. 

If Ionic supports SSR then it is in the scope of the platform to support web experiences. The majority of web traffic comes through mobile. This should be addressed.  

**Steps to reproduce:**
Our platform (PraiseCharts)[https://www.praisecharts.com] is deployed with Ionic. Try on safari or chome mobile.

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic-framework/issues/19081. The problem here is that the status bar only hides when the `body` element is scrolling. This is a Chrome/Safari limitation, not something Ionic Framework imposes. We scroll on `ion-content`, so we are affected by this limitation.

**Comment by isaiahdahl** [comment]
Thanks for responding @liamdebeasi 

Unfortunately, I don't think this is an issue that should be left to Chrome/Safari to fix. You guys made the decision to scroll on ion-content and not provide support to scroll on body somehow in the browser. 

The issue that this is a duplicate of is locked and no one has seen any updates since last year. If you're going to close this down as a duplicate, would be great if you unlocked that issue and provided an update. At least explain some of the investigating that's gone on. Have the Ionic team even explored trying to find a work around? Discussed it?

**Comment by liamdebeasi** [comment]
Yes, this is something we have discussed extensively over the course of both v4 and v5 development. The solution here is not as straightforward as it may seem. 

If we had your page's content scrolling on the `body`, certain visual aspects of your mobile application would be incorrect. For example, since everything is rendered in the `body`, your `ion-header` and `ion-footer` would not stay fixed to the top/bottom of the screen when you rubber banded (overscroll on iOS). The same goes for `ion-tab-bar`. Since these components are all in your scrollable element, there would be no way for us to fix that (barring some hacky JS solution). I put together a small demo to illustrate some of the visual bugs that appear with this approach: https://codepen.io/liamdebeasi/pen/JjWYRGw.

While moving the content onto the `body` is technically feasible, there are other issues that appear that do not have good solutions. Having the browser vendors expose an API that lets us control this scroll collapsing behavior would allow us to get the best of both worlds, which is why we are in favor of that approach.

We are aware of the issue, but we do not have a good solution at this time. We will post any updates in the thread I linked to in my previous comment.
